basics/components.md

```diff
@@ -12,15 +12,15 @@ Most of the components in GrimoireLab are written in Python, and provide Python
 
-* **Perceval**. Includes components for retrieving data from data sources. All components provide an uniform API for accessing their data sources, based on Python generators and data dictionaries (JSON documents). [Learn more about Perceval](/grimoirelab-tutorial/perceval/intro.md).
-* **Graal**. leverages on the Git backend of Perceval and enhances it to set up ad-hoc source code analysis. It fetches the commits from a Git repository and provides a mechanism to plug third-party tools/libraries focused on source code analysis. [Learn more about Graal](/grimoirelab-tutorial/graal/intro.md).
-* **KingArthur**. Organizes the data retrieval from large collections of repositories of different kinds, providing a uniform API to all of them, and to the different retrieval jobs running in any moment. [Learn more about KingArthur](/grimoirelab-tutorial/basics/scenarios.md#arthur-orchestrating-data-retrieval).
-* **GrimoireELK**. Collects data retrieved by Arthur, or directly by Perceval, and stores it in ElasticSearch indexes (raw indexes). It also organizes and transforms those raw indexes into other ElasticSearch indexes (enriched indexes), more suitable for visualization or reporting. [Learn more about GrimoireELK](/grimoirelab-tutorial/gelk/intro.md).
-* **SortingHat**. Manages identities for persons, including capabilities for merging the many identities corresponding to the same person, and other data related to them (such as affiliation to companies). [Learn more about SortingHat](/grimoirelab-tutorial/sortinghat/data.md).
+* **Perceval**. Includes components for retrieving data from data sources. All components provide an uniform API for accessing their data sources, based on Python generators and data dictionaries (JSON documents). [Learn more about Perceval](/perceval/intro.md).
+* **Graal**. leverages on the Git backend of Perceval and enhances it to set up ad-hoc source code analysis. It fetches the commits from a Git repository and provides a mechanism to plug third-party tools/libraries focused on source code analysis. [Learn more about Graal](/graal/intro.md).
+* **KingArthur**. Organizes the data retrieval from large collections of repositories of different kinds, providing a uniform API to all of them, and to the different retrieval jobs running in any moment. [Learn more about KingArthur](/basics/scenarios.md#arthur-orchestrating-data-retrieval).
+* **GrimoireELK**. Collects data retrieved by Arthur, or directly by Perceval, and stores it in ElasticSearch indexes (raw indexes). It also organizes and transforms those raw indexes into other ElasticSearch indexes (enriched indexes), more suitable for visualization or reporting. [Learn more about GrimoireELK](/gelk/intro.md).
+* **SortingHat**. Manages identities for persons, including capabilities for merging the many identities corresponding to the same person, and other data related to them (such as affiliation to companies). [Learn more about SortingHat](/sortinghat/data.md).
```

Comment:
Thanks for pointing the broken links but they have fixed differently. https://chaoss.github.io/grimoirelab-tutorial/basics/components.html

All the links should end with `.html` and `grimoirelab-tutorial/` shouldn't be deleted from the link.
It would be great if you can update the required changes.

Reply:
okay  Sir